documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc

```diff
@@ -94,6 +94,10 @@ Beware that default LIST semantics only affects owned collection mappings.
 Unowned mappings like `@ManyToMany(mappedBy = "...")` and `@OneToMany(mappedBy = "...")` do not retain the element
 order by default, and explicitly annotating `@OrderColumn` for `@ManyToMany(mappedBy = "...")` mappings is illegal.
 
+Keep in mind that the value of the expression `nullable` of `@OrderColumn` is conditionally ignored during the
+binding process as it is determined for the index column automatically.  If the index column belongs to the join
+table it is then force set to be not null, otherwise the value specified is used.
+
```

Comment:
Honestly this language isn't clear at all.

As a Hibernate user I don't know what the "binding process" is, or what it is that "conditionally ignores" the setting, nor is it clear under what conditions it's "conditionally ignored".

You're describing what the code does from an internal perspective. You're should be defining the _semantics_ of the `nullable` member.

Reply:
I would say something like:

> Order columns belonging to join tables are always non-null, and so the value of the `nullable` member of `OrderColumn` is ignored in this case.

That's much easier to understand, don't you think?

Reply:
Indeed so.  I sort of referred to annotation members as expressions as they are so named in the JSR but I'm happy with whatever the project owner would rather have it say, as long as it is said the good deed is done.  Committed changes.